contracts/0.8.9/WithdrawalQueue.sol

```diff
@@ -318,7 +320,7 @@ contract WithdrawalQueue is AccessControlEnumerable {
 
     /// @notice Returns all withdrawal requests placed for the `_recipient` address
     function getWithdrawalRequests(address _recipient) external view returns (uint256[] memory requestsIds) {
```

Comment:
maybe put a warning regarding unexpected gas spending (similar to the provided [here](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/release-v4.4/contracts/utils/structs/EnumerableSet.sol#L342:L345))